### ISSUE TITLE
I cannot open main.pyw.

### ISSUE THREAD

**Issue by damare1** [iholston/lol-bot#126]
I cannot open main.pyw.
This is a problem with ver 2.0.0.
I can run it on my desktop PC with no problem, but I cannot open main.pyw on my laptop.(I get no error messages. I clicked on it and got no response.)
It works fine with ver 1.3.1. What could be the cause?
Both desktop and laptop computers have windows 10 64 bit OS.
Since version 2.0.0, do I have to install something else to get it to work?
I'm using Ver 1.3.1 now until this is resolved.

**Comment by iholston** [comment]
you will need to rerun installer.bat as there are new dependencies

**Comment by damare1** [comment]
I have run installer.bat but cannot open main.pyw.

**Comment by iholston** [comment]
open cmd and navigate to the folder where the bot is then run 

python main.pyw

and it should give you an error, paste that in here.

**Comment by damare1** [comment]
![err](https://github.com/iholston/lol-bot/assets/144917228/7dfd8ea8-b6af-435e-b0a3-b9f94af9dcb6)
I tried running it with cmd, but the result was the same.
I cannot open it.
Ver 1.3.1 can be executed without any problem, but I wonder why...

**Comment by damare1** [comment]
![123](https://github.com/iholston/lol-bot/assets/144917228/c3bcf842-b779-4df3-b804-a1046f577c6f)
cd C:\Users\user\Desktop\lol-bot-2.0.0
python main.pyw
After entering the above, I got this message. Do you know how to resolve this?
【指定されたモジュールが見つかりません】＝The specified module cannot be found. Meaning.

**Comment by iholston** [comment]
Oh wow yea. So this is related to this issue: https://github.com/hoffstadt/DearPyGui/issues/298

The solution (which I will have to add to the Readme) is to go here: https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads and download the latest package for your architecture. 

TLDR: click and install this: https://aka.ms/vs/17/release/vc_redist.x64.exe

**Comment by damare1** [comment]
![da23](https://github.com/iholston/lol-bot/assets/144917228/e2ba3220-c4f4-4e6f-bcc4-cdba3ae9d3aa)
Thanks. I installed it and was able to open it.
